packages/vector_graphics/lib/src/listener.dart

```diff
@@ -351,8 +446,11 @@ class FlutterVectorGraphicsListener extends VectorGraphicsCodecListener {
   }
 
   @override
-  void onDrawText(int textId, int paintId) {
+  void onDrawText(int textId, int paintId, int? patternId) async {
     final ui.Paint paint = _paints[paintId];
+    if (patternId != 65535) {
```

Comment:
here too, what is this number?

Reply:
ahh, -1 as an unsigned int.

Reply:
make this a constant and give it a descriptive name. We call this kMaxId in the codec path.

Also: rather than passing the sentinel value through to this method, you should detect that the value is the sentinel in the decoder and pass null instead

Reply:
fixed.